src/SB/Core/x/xSpline.h

```diff
@@ -8,6 +8,10 @@ struct xCoef
     F32 a[4];
 };
 
+static F32 sBasisUniformBspline[4][4];
+static F32 sBasisBezier[4][4];
+static F32 sBasisHermite[4][4];
```

Comment:
Did you have a reason for putting these into the header instead of the cpp file?